app/src/main/kotlin/com/hippo/ehviewer/ui/MainActivity.kt

```diff
@@ -345,7 +346,7 @@ class MainActivity : EhActivity() {
                                                 },
                                                 modifier = Modifier.padding(horizontal = 12.dp),
                                                 icon = {
-                                                    Icon(imageVector = icon, contentDescription = null)
+                                                    Icon(painter = rememberNeverRecomposeVectorPainter(icon), contentDescription = null)
```

Comment:
Idk if this is a good idea, seems like recomposition is caused by `currentDestination`.

Reply:
Nope, root cause is VectorPainter infinitely invalidate itself when recompose, this also happens in galleryscreen

Reply:
But it shouldn't recompose that often?

Reply:
I mean, once rememberImageVector is recomposed
This https://github.com/androidx/androidx/blob/383259a3318443cad02748c73d38423ac2c00405/compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/graphics/vector/VectorPainter.kt#L237 will triggered
And setName invalidate itself, https://github.com/androidx/androidx/blob/383259a3318443cad02748c73d38423ac2c00405/compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/graphics/vector/VectorPainter.kt#L196 invalidateCount++, mutableIntState changed, trigger next recompose
LMAO

Reply:
The only thing we can do is prevent rememberImageVector being recomposed